model/shared/src/main/scala/explore/model/Perishable.scala

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.Eq
+
+// A value, A, that can be marked as stale or errored.
+opaque type Perishable[A] = (A, Boolean)
```

Comment:
so the bool is true/false for stale/errored? or false means either stale or errored?

Reply:
Yes, true is stale. But, that isn't visible outside the type so it doesn't really matter. Users of the class use `isStale` and `setStale`.

Reply:
Maybe we can use a `NewBoolean` ?

Reply:
It's OK then since it's not visible from outside.